module-code/app/securesocial/core/OAuth2Provider.scala

```diff
@@ -160,8 +162,10 @@ trait OAuth2Provider extends IdentityProvider with ApiSupport {
           cacheService.set(sessionId, state, 300).map {
             unit =>
               val url = client.navigationFlowUrl(routesService.authenticationUrl(id), state)
-              logger.debug("[securesocial] redirecting to: [%s]".format(url))
-              AuthenticationResult.NavigationFlow(Results.Redirect(url).withSession(request.session + (IdentityProvider.SessionId -> sessionId)))
+              logger.warn("[securesocial] redirecting to: [%s]".format(url))
+              AuthenticationResult.NavigationFlow(Results.Redirect(url)
+                .withSession(request.session + (IdentityProvider.SessionId -> sessionId))
```

Comment:
We don't need this line since the data is written to cookies, do we?

Reply:
I will remove this line in forthcoming commits.